spec/acceptance_spec.rb

```diff
@@ -42,6 +42,7 @@ def read_example(name)
 
   def deliver_example(name, options={})
     deliver(read_example(name), options)
+    navigate
```

Comment:
This breaks a fairly core part of the test — that the interface should show new deliveries *without a refresh*